js/google_ima.js

```diff
@@ -1191,6 +1229,12 @@ require("../html5-common/js/utils/utils.js");
               }, this), 100);
             }
             break;
+          case eventType.PAUSED:
+            if (this.videoControllerWrapper)
```

Comment:
I don't pause the timerupdate when we get a PAUSED event in case the playhead changes while paused (such as for scrubbing). This ensures that while the video is on screen (between STARTED and COMPLETED), the playhead time is correct.